configs/typescript.example.ts

```diff
@@ -1,5 +1,6 @@
 // Use --report-unused-disable-directives to validate errors
 
+/* eslint-disable no-unused-expressions */
```

Comment:
@amacneil This one stumped me – if I remove it I get errors from `npm run lint:base`, but the `typescript.js` config should already disable this rule (because it enables the typescript-eslint version instead)... Is something weird happening because we are explicitly passing ` --config ./configs/base.js` instead of letting eslint find the config file itself?

Reply:
It looks like you added typescript to eslintrc.js - was that intentional? Pretty sure the way I set these tests up was to not include it in the base, and explicitly require it when running the typescript tests only.

Reply:
Why is it set up that way? It's normal for other packages (e.g. studio) to use eslintrc to specify the configs being imported/used, with an override rule to bring in the typescript config for .ts files, so I'm not sure why this package does it differently for internal self-linting.